### ISSUE TITLE
not working on sketch 50

### ISSUE THREAD

**Issue by bukinoshita** [sureskumar/sketch-isometric#25]
I'm using sketch version 50, would like to know if it works on v50. I try to generate the isometric but nothing happens.

**Comment by frankieGP** [comment]
2nd that. Same issue.

**Comment by dH3s1** [comment]
Yes this definitely isn't generating anything in sketch v50. Hopefully this can be resolved soon.
Thanks.

**Comment by sureskumar** [comment]
I will look into this today or tomorrow. Sorry for the delay guys!

**Comment by sureskumar** [comment]
@gokul0 Looks like this part of the code is not working in Sketch v50.

` var path = NSBezierPath.bezierPath();
    path.moveToPoint(NSMakePoint(x1, y1));
...
    path.lineToPoint(NSMakePoint(x4, y4));
    path.closePath();

    var shape = MSShapeGroup.shapeWithBezierPath(path);
    var border = shape.style().addStylePartOfType(0);`

If you have time, can you please look into this? Cheers!

**Comment by gokul0** [comment]
From what I just saw, I think it pops up because we call a method that does not exist on the object. Will have to look into it.

**Comment by sureskumar** [comment]
Thanks Gokul. The bug is now fixed in the latest version. Sorry for the inconvenience guys!

**Comment by gokul0** [comment]
@sureskumar What was the issue?

**Comment by sureskumar** [comment]
Replacing this 
`var shape = MSShapeGroup.shapeWithBezierPath(path);`
with this 
```
var newBezier = MSPath.pathWithBezierPath(path);
var shape = MSShapeGroup.shapeWithBezierPath(newBezier);
```
solved it :)

**Comment by gokul0** [comment]
I saw that in the commit. But why did we have to make this change? The old method invocation of 
```
bezierPath ()
```
 on the 
```
NSBezierPath
``` object was removed or deprecated?

**Comment by sureskumar** [comment]
To be honest, I dont understand code in that level of depth :) I saw another plugin working in sketch 50 using this and I tried it :)

**Comment by gokul0** [comment]
No probs, why is this now reopened?

**Comment by leonardofed** [comment]
Can't make this work with Sketch 49. Any updates?
